src/App.vue

```diff
@@ -1,17 +1,21 @@
 <template>
   <Heading />
+  <Introduction />
   <Dashboard />
+
 </template>
 
 <script>
 import Dashboard from './components/Dashboard.vue'
 import Heading from './components/Heading.vue'
+import Introduction from './components/Introduction.vue'
 
 export default {
   name: 'App',
   components: {
     Heading,
     Dashboard,
+    Introduction,
```

Comment:
Nice! Creating modules like this is critical to making things readable, not clunky and easy to find!